doc/source/installation.rst

```diff
@@ -44,6 +44,20 @@ if selinux is not enabled or is configured to be permissive.
 It is your reponsability to assure that soft dependencies of Ansible are
 available on your controller or host machines.
 
+.. warning::
```

Comment:
Includes a commit that shouldn't be there?

Reply:
@decentral1se this if f... weird! I have no idea where this code came from but I have the impression that somehow github combine my change with another change made by you related to setuptools. I guess you used the same branch name `fix/setuptools`?

When I pushes there was not such branch open, still the final result is a combined change. Not sure what cause it. My change was a single line and this is what I was expecting to see raise.

I will fix it now, hopefully it does not become a habit of me raising random code.

Reply:
That's one of many good reasons to open PRs from forks, I think.